package.json

```diff
@@ -16,7 +17,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "start:silent": "PORT=3001 BROWSER=none react-scripts start",
-    "cy:open": "cypress open",
+    "cy:open": "yarn start --silent & cypress open",
```

Comment:
You don't need this, use `yarn cypress` to run cypress

Reply:
Got it, will remove it!